index.html

```diff
@@ -222,58 +227,57 @@ <h1 class="mobile-about-tit">About</h1>
         </div>
         <!-- end about mobile version -->
 
-        <section class="about-page">
-        <div id="about-link" class="grid-container">
-            <div class="grid-item"> 
-                <div class="techprof">
-                    <div class="languages">
-                        <h3>Languages</h3>
-                        <ul>
-                            <li>JavaScript (ES5/ES6)</li>
-                            <li>Ruby</li>
-                            <li>HTML</li>
-                            <li>CSS</li>
-                        </ul>
+        <div class="about-page">
+            <div id="about-link" class="grid-container">
+                <div class="grid-item"> 
+                    <div class="techprof">
+                        <div class="languages">
+                            <h3>Languages</h3>
+                            <ul>
+                                <li>JavaScript (ES5/ES6)</li>
+                                <li>Ruby</li>
+                                <li>HTML</li>
+                                <li>CSS</li>
+                            </ul>
+                        </div>
```

Comment:
- [ ] Please refer to the figma to implement the following:

   ------------------------Original----------------------
![header3](https://user-images.githubusercontent.com/55635977/126672547-be2eeb9d-746a-429c-a18b-a0f5c7c6d3dc.png)


  -------------------------Yours------------------------ 
![header4](https://user-images.githubusercontent.com/55635977/126672267-c11c9825-0c84-441c-82d2-3591b71dca04.png)

Reply:
>     * [ ]  Please refer to the figma to implement the following:
>       ------------------------Original----------------------
>       ![header3](https://user-images.githubusercontent.com/55635977/126672547-be2eeb9d-746a-429c-a18b-a0f5c7c6d3dc.png)
>       -------------------------Yours------------------------
>       ![header4](https://user-images.githubusercontent.com/55635977/126672267-c11c9825-0c84-441c-82d2-3591b71dca04.png)